apps/examples/App.tsx

```diff
@@ -107,6 +112,10 @@ const SCREENS: Record<
   },
 };
 
+const screens = Object.keys(SCREENS).filter(
```

Comment:
I wonder if it wouldn't be clearer to just do some method that returns check for each platform. What do you think?